gitops/addons/charts/adot-collector/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: opentelemetry
+description: A Helm chart to install otel operator
+type: application
+version: 0.8.0
```

```suggestion
version: 1.0.0
```
Since its first version, lets use `1.0.0`